test/ProductionModePlugin.js

```diff
@@ -96,7 +96,7 @@ function commonTests(testName, webpackConfig, outputPath) {
       require(mkOutputPath(testName, "app"));
 
       const globalizeModuleStats = compileStats.toJson().modules.find((module) => {
-        return module.name === "./~/globalize/dist/globalize-runtime.js";
+        return module.name === "./node_modules/globalize/dist/globalize-runtime.js";
```

Comment:
This change causes test failure... The "home tilde" `~`here seems like a webpack thing.

Reply:
It seems like another change in webpack 3 then :)